components/AlertCard/AlertCardBody.tsx

```diff
@@ -1,10 +1,18 @@
 import { ReactElement } from "react"
 import CardBootstrap from "react-bootstrap/Card"
+import styled from "styled-components"
+import { Col, Row, Spinner } from "../bootstrap"
+import { Timestamp } from "firebase/firestore"
+
+const AlertCardInnerBodyV2 = styled.div`
+  background-color: #d1d6e7;
```

Comment:
Going to be the same nit on all of these `V2` components - if they're just `NewsfeedCard*`, let's just name them that to be unambiguous. The props are pretty tied to the newsfeed data model anyway.

Reply:
Also nit: Can this background-color be one of our color variables? I think we're looking to keep these more consistent going forward.